drivers/slipdev/slipdev.c

```diff
@@ -67,11 +68,90 @@ static inline void slipdev_unlock(void)
     }
 }
 
-static void _slip_rx_cb(void *arg, uint8_t byte)
+static inline void _slipdev_stdio_add_to_frame(slipdev_t * dev, uint8_t byte) {
+    if (IS_USED(MODULE_SLIPDEV_STDIO)) {
+        if (dev->config.uart == slipdev_params[0].uart) {
```

Comment:
Not really relevant here, but generally keeping the indentation level low helps with readability.


```suggestion
    if (!IS_USED(MODULE_SLIPDEV_STDIO)) ||
        dev->config.uart != slipdev_params[0].uart) {
        return;
    }
```